src/99-appendices/10-file-collections.md

```diff
@@ -37,11 +43,18 @@ included in this appendix:
-| TB1RMF | Please see the qMRI appendix | Siemens `rf_map`                   | B1<sup>+</sup> data acquired using `rf_map` product sequence by Siemens                                                                                                                                                                                                                                                                                                           |
-| RB1COR | Please see the qMRI appendix | B1<sup>-</sup> field correction    | Low resolution images acquired by the body coil (in the gantry of the scanner) and the head coil using identical acquisition parameters to generate a combined sensitivity map as described in [Papp et al. (2016)](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.26058)                                                                                                   |
+```Text
+sub-<label>/[ses-<label>/]
+    fmap/
+        sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>][_rec-<label>][_run-<index>][_echo-<index>][_flip-<index>][_inv-<index>][_mt-<label>][_part-<label>]_<suffix>.nii[.gz]
+```
+
+| Suffix  | Meta-data relevant entity                                    | Application                        | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+|---------|--------------------------------------------------------------|------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| TB1DAM  | flip                                                         | Double-angle B1+ mapping           | The double-angle B1<sup>+</sup> method ([Insko and Bolinger 1993](https://www.sciencedirect.com/science/article/abs/pii/S1064185883711332)) is based on the calculation of the actual angles from signal ratios, collected by two acquisitions at different nominal excitation flip angles. Common sequence types for this application include spin echo and echo planar imaging.                                                                                                |
```

Comment:
please replace all links to papers with a DOI url like `https://doi.org/<put DOI here>`

Reply:
https://github.com/bids-standard/bids-specification/pull/690/commits/0bda72f8ada0eac968406f82018633a7957f700a